ceph.spec.in

```diff
@@ -134,6 +134,9 @@ BuildRequires:	snappy-devel
 BuildRequires:	udev
 BuildRequires:	util-linux
 BuildRequires:	valgrind-devel
+%if %{with make_check}
+BuildRequires:	which
+%endif
```

Comment:
@b-ranto @tchaikov I'm tempted to remove the conditional here and just build-require which, even though strictly speaking it's only needed for the tests. What do you think?

I'm not sure how to pass `--with make_check` to `rpmspec` in `install-deps.sh` and it seems like it's not even worth the effort.

(Background: we discovered this issue by running "make check" in a rather pristine Cloud environment - it's rare to find a "real" machine that doesn't have which installed)